### PR TITLE
fix: update database schema

### PR DESCRIPTION
**Pull request by Icemap**
Movie schema update by document [1. Database Schema Design.md](https://github.com/pingcap-inc/tidb-developer-guide/blob/main/12.%20Reference/3.%20SQL/1.%20The%20Movie%20Example%20Application/1.%20Database%20Schema%20Design.md)

### CODE REVIEW

File: movie/ddl.go [hunk 1]
```diff
@@ -43,12 +43,12 @@ func (w *ddlManager) createTables(ctx context.Context) error {
...
-			title VARCHAR(128),
-			type VARCHAR(20),
-			year SMALLINT,
-			released_at DATETIME,
-			PRIMARY KEY (id)
+			id bigint(20) NOT NULL,
+			title varchar(100) NOT NULL,
+			type enum('Action','Adventure','Animation','Children''s','Comedy','Crime','Documentary','Drama','Fantasy','Film-Noir','Horror','Musical','Mystery','Romance','Sci-Fi','Thriller','War') NOT NULL,
+			year smallint(6) NOT NULL,
+			release_time datetime NOT NULL,
```
Comment (Mini256) [inline movie/ddl.go]: Considering that the rating time is suffixed with `at`(`rating_at`), I want to unify this naming style.

File: movie/load.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ var movieTypes = []string{
 	"Action",
 	"Adventure",
 	"Animation",
-	"Children",
+	"Children's",
```
Comment (Mini256) [inline movie/load.go]: It also needs to be modified at the same time. Maybe we need to define a set of constants.

https://github.com/Mini256/tidb-dataset/blob/cbcc9150b061088db6e4521ccc6d49677bdf8123/movie/load.go#L109

File: movie/ddl.go (hunk 1 above)
Comment (Icemap) [reply movie/ddl.go]: OK

**Review by Mini256 — APPROVED** [review]

**Comment by Mini256** [comment]
@Icemap Thank you for your Pull Request!

**Pull request merged**